JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -100,9 +104,13 @@ fun PostContent(modifier: Modifier = Modifier.None, post: Post) {
 @Composable
 private fun PostHeaderImage(post: Post) {
     post.image?.let { image ->
-        Container(modifier = LayoutHeight.Min(180.dp) + LayoutWidth.Fill) {
+        val sizingModifier = LayoutHeight.Min(180.dp) + LayoutWidth.Fill
```

Comment:
Why is this referenced twice? This should only be needed on on the outer Box, right?

Reply:
Yes, there's some issues with layout in dev06 that make this needed – in this case the image isn't properly expanding to the Clip region without specifying the size twice.

This is expected to be resolved shortly when the Clip modifier and Image component are added.

Reply:
Can we add a clearer comment then? 😄 Also make it a `FIXME`

Reply:
Yep! Updating

Reply:
Done!